code/src/NCModel/Features.ts

```diff
@@ -878,7 +598,7 @@ export class NcReceiverMonitorProtected extends NcReceiverMonitor
     }
 }
 
-enum NcDemoEnum
+enum DemoEnum
```

Comment:
Similar to comment above, given this is a "vendor specific" Enum, could it be renamed to VendorSpecificEnum perhaps?

Reply:
We could altough there is no requirement for vendors to name custom classes in any particular way.

Reply:
In line with the above I will change this to something like "ExampleEnum".